packages/babel-core/src/gensync-utils/fs.ts

```diff
@@ -10,12 +10,7 @@ export const readFile = gensync<(filepath: string, encoding: "utf8") => string>(
 
 export const exists = gensync<(filepath: string) => boolean>({
   sync(path) {
-    try {
-      fs.accessSync(path);
-      return true;
-    } catch {
-      return false;
-    }
+    return fs.existsSync(path);
   },
   errback: (path, cb) => fs.access(path, undefined, err => cb(null, !err)),
```

Comment:
```suggestion
  errback: (path, cb) => fs.access(path, undefined, err => cb(null, !err)),
```

We can delete his so that it always uses `fs.existsSync` (both in sync and async mode). By doing so we don't need to worry about possible behavior differences.

Reply:
Agreed.